src/main/java/Main.java

```diff
@@ -1,20 +1,33 @@
 import UI.*;
 import config.CommonCryptograph;
 import config.Cryptograph;
+import config.UserDataStoreGateway;
 import config.UserStorage;
-import entity_layer.CommonUserFactory;
+import entity_factories.CommonUserFactory;
+import entity_layer.UserFactory;
+import use_cases.login.LoginController;
+import use_cases.login.LoginInputBoundary;
+import use_cases.login.LoginInteractor;
+import use_cases.user_registration.UserRegistrationController;
 import use_cases.user_registration.UserRegistrationInteractor;
 
 import javax.swing.*;
 
 public class Main {
```

Comment:
Thanks so much for setting this up! Extracting the login into a Main driver will definitely help clean up our architecture and make the dependencies a lot more fluid.